### ISSUE TITLE
Concept Set Annotation / Attachment Option #enhancement

### ISSUE THREAD

**Issue by mattlevine22** [OHDSI/Circe#43]
Since unique choices and circumstances are often involved in creating a concept set, it may be helpful  to be able to include a description and, perhaps even better, attach a document that describes how this concept set was defined.

**Comment by jduke99** [comment]
I agree with Matt but would take it a step further and say that we need to think about our Knowledge Library as a whole.  What metadata should we be capturing around concept sets and cohort definitions?  I think our Knowledge Library / Content Library may merit a standalone WG effort to plan the right strategy for this.
